entity-framework/core/managing-schemas/migrations/index.md

```diff
@@ -1,294 +1,184 @@
+* Once a new migration has been generated, it can be deployed to a database in various ways. EF Core uses a migrations history table in the database which records all migrations which have already been applied; this allows EF Core to know which state your database is in.
 
-> [!TIP]
-> If the `DbContext` is in a different assembly than the startup project, you can explicitly specify the target and startup projects in either the [Package Manager Console tools](xref:core/miscellaneous/cli/powershell#target-and-startup-project) or the [.NET Core CLI tools](xref:core/miscellaneous/cli/dotnet#target-project-and-startup-project).
+## Tutorial
 
-## Install the tools
+Let's assume you've just completed your first EF Core application, which contains the following simple model:
 
-Install the [command-line tools](xref:core/miscellaneous/cli/index):
-
-* For Visual Studio, we recommend the [Package Manager Console tools](xref:core/miscellaneous/cli/powershell).
-* For other development environments, choose the [.NET Core CLI tools](xref:core/miscellaneous/cli/dotnet).
-
-## Create a migration
-
-After you've [defined your initial model](xref:core/modeling/index), it's time to create the database. To add an initial migration, run the following command.
-
-### [.NET Core CLI](#tab/dotnet-core-cli)
-
-```dotnetcli
-dotnet ef migrations add InitialCreate
-```
-
-### [Visual Studio](#tab/vs)
-
-``` powershell
-Add-Migration InitialCreate
+```c#
+public class Blog
+{
+    public int Id { get; set; }
+    public string Name { get; set; }
+}
 ```
 
-***
-
-Three files are added to your project under the **Migrations** directory:
+During development, you may have used the [Create and Drop APIs](xref:core/managing-schemas/ensure-created) to iterate quickly; but now that your application is going to production, you need a way to evolve the schema without dropping the entire database.
 
-* **XXXXXXXXXXXXXX_InitialCreate.cs**--The main migrations file. Contains the operations necessary to apply the migration (in `Up()`) and to revert it (in `Down()`).
-* **XXXXXXXXXXXXXX_InitialCreate.Designer.cs**--The migrations metadata file. Contains information used by EF.
-* **MyContextModelSnapshot.cs**--A snapshot of your current model. Used to determine what changed when adding the next migration.
+### Install the tools
 
-The timestamp in the filename helps keep them ordered chronologically so you can see the progression of changes.
+First, you'll have to install the [EF Core command-line tools](xref:core/miscellaneous/cli/index):
 
-### Namespaces
+* We generally recommend using the [.NET Core CLI tools](xref:core/miscellaneous/cli/dotnet), which work on all platforms.
```

Comment:
Is it fair to say we recommend the CLI over PMC as a general rule, unless otherwise? That seems to make sense, from a cross-platform cross-IDE standpoint.

Reply:
Also, "which works"

Reply:
Should it be plural because we speak of CLI tools?